src/_includes/components/contact.pug

```diff
@@ -6,18 +6,14 @@ section.contact#contact
       dl
         dt Call
         dd
-          a(href="tel:{{site.contact.tel}}") {{ site.contact.tel }}
+          a(href="tel:+1.{{site.contact.tel}}") {{ site.contact.tel }}
```

Comment:
Thanks! Except this points out a bigger flaw. Will fix on my end after this is merged in.